src/validations/tripValidation.js

```diff
@@ -0,0 +1,33 @@
+import DateExtension from '@joi/date';
+import * as JoiImport from 'joi';
+
+const Joi = JoiImport.extend(DateExtension);
+
+const staticsValidate = Joi.object().keys({
+  start: Joi.date()
+    .utc('javascript')
```

Comment:
This is what I mean, UTC allows different formats. eg DD-MM-YYYY. which is what failed in my PR comment above